lib/api/src/Cardano/Wallet/Api/Types/Error.hs

```diff
@@ -159,6 +160,8 @@ data ApiErrorInfo
         !ApiErrorNoSuchPool
     | NoSuchTransaction
     | NoSuchWallet
+        !ApiErrorNoSuchWallet
+    | NoSuchWalletInitialized
```

Comment:
If this error is a **_known_** wallet that is simply not initialized, then can we use a name like:
```suggestion
    | WalletNotInitialized
```

Reply:
sounds reasonable - done